client/reader/discover/discover-stream.js

```diff
@@ -49,6 +49,12 @@ const DiscoverStream = ( props ) => {
 		},
 	} );
 
+	// Add dailyprompt to the front of interestTags if not present.
+	const hasDailyPrompt = interestTags.filter( ( tag ) => tag.slug === 'dailyprompt' ).length;
+	if ( ! hasDailyPrompt ) {
+		interestTags.unshift( { title: translate( 'Daily prompts' ), slug: 'dailyprompt' } );
+	}
```

Comment:
Two relevant things for this component:
1. We shouldn't be importing `translate` directly -- we should be importing `useTranslate` and then setting `const translate = useTranslate();` towards the top of the component logic.
2. For the actual tests, we can make use of the `useIsEnglishLocale()` and/or `useHasEnTranslation()` hooks from `i18n-utils` to identify whether a specific string is translated (or can be left as-is because we're in an English locale).

Reply:
> We shouldn't be importing translate directly -- we should be importing useTranslate and then setting const translate = useTranslate(); 

Oof yes thank you. I caught this in trying to implement the above I mentioned which I didn't commit, but forgot to bring it in here.